### ISSUE TITLE
Bad signature is produced for function pointers

### ISSUE THREAD

**Issue by tamasvajk** [JetBrains/Refasmer#20]
`delegate*<void>` doesn't seem to be handled correctly.

Taking the below code:
```
public unsafe class X
{
    public void M(delegate*<void> p) { }
}
```
and calling `refasmer X.dll -v -c -p -n -q -o=Y.dll` produces an assembly that has `<bad signature>` in the method `M`:
```
.class /* 02000004 */ public auto ansi beforefieldinit X
	extends [System.Runtime]System.Object
{
	// Methods
	.method /* 06000003 */ public hidebysig 
		<bad signature> M () cil managed 
	{
	} // end of method X::M

	.method /* 06000004 */ public hidebysig specialname rtspecialname 
		instance void .ctor () cil managed 
	{
	} // end of method X::.ctor

} // end of class X

```

**Comment by zabbius** [comment]
The fix is here

https://github.com/JetBrains/Refasmer/pull/21

**Comment by ForNeVeR** [comment]
Closed by #21.
